src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,170 @@
 package core.basesyntax;
 
+//import jdk.internal.util.ArraysSupport;
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+
+
 public class ArrayList<T> implements List<T> {
+
+
+    private static final int def_capacity = 10;
```

Comment:
Pls rename all variables according to name convention https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html
```java
private static final int defCapacity = 10;
```